airflow/providers/amazon/aws/operators/redshift_to_s3.py

```diff
@@ -101,13 +106,19 @@ def __init__(  # pylint: disable=too-many-arguments
             self.unload_options = list(self.unload_options) + ['HEADER', ]
 
     def execute(self, context):
+        if all([self.schema, self.table]):
```

Comment:
```suggestion
        if self.schema and self.table:
```
Is `all` really necessary?

Reply:
Not really

Reply:
I think `all` makes it cleaner.

Reply:
Please use `and`. `all` requires a list which you do not need if you use `and`.